MyFirstSlideshow/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,11 +4,87 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.charles.myfirstslideshow.MainActivity">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        />
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="400px"
+            android:id="@+id/image1"/>
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+            <View
+                android:id="@+id/dots1dot1"
+                android:layout_width="100px"
+                android:layout_height="20px"
+                android:layout_weight="1"
+                android:background="#008000"/>
```

Comment:
activity_main : line 24 - color code should be defined from color.xml

Reply:
activity_main : line 21 - if using weight in any view. you should make android:layout_width= "0dp"
 activity_main : line 24 - colors should be declare in colors.xml file in res folder.